### PR TITLE
Rename plugin->extension and GITServices->GitHubExtension

### PR DESCRIPTION
**Pull request by EricJohnson327**
## Summary of the pull request
Renaming all instances of the following:
"plugin" -> "extension"
"GITServices" -> "GitHubExtension"

## PR checklist
- [ ] Closes #xxx
- [ ] Tests added/passed
- [ ] Documentation updated

### CODE REVIEW

File: build/scripts/CertSignAndInstall.ps1 [hunk 1]
```diff
@@ -9,7 +9,7 @@ function Invoke-SignPackage([string]$Path) {
         return
     }
 
-    $certName = "Microsoft.GITServices"
+    $certName = "Microsoft.GitHubExtension"
```
Comment (vineeththomasalex) [inline build/scripts/CertSignAndInstall.ps1:12]: Do we need to add this file or is it coming from the pipeline?

**Review by dhoehna — APPROVED** [review]

**Review by AmelBawa-msft — APPROVED** [review]

**Review by vineeththomasalex — APPROVED** [review]
Just wanted to confirm that this doesn't change anything that can affect the package identity, since that will affect the Store listing.

**Review by krschau — APPROVED** [review]
Cloned, built, and all widgets still seem to work!

File: build/scripts/CertSignAndInstall.ps1 (hunk 1 above)
Comment (EricJohnson327) [reply build/scripts/CertSignAndInstall.ps1:12]: This is generated for you when building locally

**Pull request merged**